skills/gh-gantt-workflow/SKILL.md

```
@@ -0,0 +1,107 @@
+gh-gantt pull
+gh issue list
+gh issue view <number>
+```
+
+Optionally visualize with `gh-gantt serve` (opens Gantt chart at http://localhost:3000).
+
+### 2. Branch
+
+Create a feature branch from the issue:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)